decidim-budgets/app/packs/src/decidim/budgets/admin/projects.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-invalid-this */
+$(() => {
+  const selectedProjectsCount = function() {
```

Comment:
I tried to merge code in ```projects.js``` and ```proposals.js``` but not sure how we could override javascript functions with current solution. Also we would need to change selectors in proposal views or maybe api functions could have css selector params. However I think it would be outside of the scope of this pull request and should be another task.